### DISCUSSION TITLE
Missing Features on Tab S7 FE

### DISCUSSION THREAD

**Discussion by Beerusqma** [tribalfs/GalaxyMaxHzPub#222] [Q&A]
I have a question why my GMZ Dont have other Future like others it have only:
PSM
Auto disable sync
Quick Doze mod
Protect battery mod 
Animation mod
Net speed indicator?

https://github.com/tribalfs/GalaxyMaxHzPub/assets/175297052/67ecc6cf-16cb-4493-81af-6acc1442d689

**Comment by tribalfs** [comment]
Specify your device

**Comment by Beerusqma** [comment]
Galaxy Tab s7 fe

**Reply by tribalfs** [reply]
If you are looking for refresh rate related features, it's because this device's display doesn't support high refresh rates.

**Comment by Beerusqma** [comment]
And then the others features

**Reply by tribalfs** [reply]
Read app description. 
That will be hard to answer without specifying which feature(s) you're referring to.

**Comment by Beerusqma** [comment]
What?Sent from my iPhoneOn Sep 30, 2024, at 6:25 AM, Tribalfs ***@***.***> wrote:﻿
Closed #222 as resolved.
